client/package.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
- `parcel` compiles the stuff in parcel-entry.js to Django's static ---> `../server/mapper/static`. 
- The output files are also named `parcel-entry.js`, I assume there's some way to change but just rolling with it